### PR TITLE
Planetary Starlane Drive - Beacon placement limit & Encyclopedia Text

### PR DESCRIPTION
**Pull request by rrobgill**
Planetary beacon construction prohibited in systems with no free
orbits.

Text added to encyclopedia about system size limit of 7 planets for
use of beacon placement.

Text added describing outcome if attempting to move planet into a
system with no available orbits (eg via Beacon ship part or
attempting to move multiple planets in a single turn).

(Related bug #917)

Signed-off-by: Rob Gill <rrobgill@protonmail.com>

### CODE REVIEW

File: default/scripting/buildings/PLANET_BEACON.focs.txt [hunk 1]
```diff
@@ -19,4 +24,5 @@ BuildingType
     icon = "icons/building/beacon.png"
 
 #include "/scripting/common/enqueue.macros"
-#include "/scripting/common/base_prod.macros"
\ No newline at end of file
+#include "/scripting/common/base_prod.macros"
```
Comment (geoffthemedio) [inline default/scripting/buildings/PLANET_BEACON.focs.txt:27]: is this `#include` needed? it doesn't seem relevant to the script additions.

File: default/stringtables/en.txt [hunk 2]
```diff
@@ -13291,13 +13291,16 @@ BLD_PLANET_DRIVE_DESC
...
 
-To use this building, either construct a [[buildingtype BLD_PLANET_BEACON]] in the adjacent destination system or move your ship with a [[shippart SP_PLANET_BEACON]] there, then set the planet to Planet Drive focus. The planet will then be moved to that system. Note the beacon needs to have been in system for a turn: the drive will not work on the turn the building is completed or on the turn the ship arrives. Also note, a [[buildingtype BLD_TRANSFORMER]] can be used to replicate the effects of this building.'''
+To use this building, either construct a [[buildingtype BLD_PLANET_BEACON]] in the adjacent destination system or move your ship with a [[shippart SP_PLANET_BEACON]] there, then set the planet to Planet Drive focus. The planet will then be moved to that system. Note the beacon needs to have been in system for a turn: the drive will not work on the turn the building is completed or on the turn the ship arrives. Also note, a [[buildingtype BLD_TRANSFORMER]] can be used to replicate the effects of this building.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: avoid referring to the player in the second person: "you" / "your"

File: default/scripting/buildings/PLANET_BEACON.focs.txt (hunk 1 above)
Comment (rrobgill) [reply default/scripting/buildings/PLANET_BEACON.focs.txt:27]: The include was in the file to begin with. I think there was a newline appended causing it to be highlighted? Removed.

Comment (geoffthemedio) [reply default/scripting/buildings/PLANET_BEACON.focs.txt:27]: probably needed then (though ideally without the whitespace noise)
